app/docs/0.13.x/admin-api.md

```diff
@@ -2253,6 +2253,75 @@ HTTP 204 No Content
 
 ---
 
+### Show upstream health for node
+
+Displays the health status for all targets of a given upstream, according to
```

Comment:
could we capitalize "targets" and "upstream" here and in other occurrences? We try to capitalize Kong entities through-ought the docs and I think we should stay consistent

Reply:
Yes please, good catch @thibaultcha